python/cable_cell_io.cpp

```diff
@@ -9,14 +9,18 @@
 #include <arborio/cableio.hpp>
 
 #include "error.hpp"
+#include "util.hpp"
 #include "strprintf.hpp"
 
 namespace pyarb {
 
-arborio::cable_cell_component load_component(const std::string& fname) {
+namespace py = pybind11;
+
+arborio::cable_cell_component load_component(py::object fn) {
+    const auto fname = util::to_path(fn);
     std::ifstream fid{fname};
     if (!fid.good()) {
-        throw pyarb_error("Can't open file '{}'" + fname);
+        throw pyarb_error(util::pprintf("Can't open file '{}'", fname));
```

Comment:
I'd be happy to have a new-style typed error here, but I'll leave that up to you.

Reply:
That's a good point, I should remember my own demands.